doc/man3/OPENSSL_init_crypto.pod

```diff
@@ -190,15 +190,20 @@ OPENSSL_init_crypto(). For example:
 
 The OPENSSL_cleanup() function deinitialises OpenSSL (both libcrypto
 and libssl). All resources allocated by OpenSSL are freed. Typically there
-should be no need to call this function directly as it is initiated
-automatically on application exit. This is done via the standard C library
-atexit() function. In the event that the application will close in a manner
-that will not call the registered atexit() handlers then the application should
-call OPENSSL_cleanup() directly. Developers of libraries using OpenSSL
-are discouraged from calling this function and should instead, typically, rely
-on auto-deinitialisation. This is to avoid error conditions where both an
-application and a library it depends on both use OpenSSL, and the library
-deinitialises it before the application has finished using it.
+should be no need to call this function directly as it is done 
+automatically on application exit. This is arranged by the linker which ensures
+a library destructor gets invoked on library unload (or process exit).  On some
+platforms (or if your application is linked statically) there is no support for
```

Comment:
again the static linking thing seems to be not correct generically

Reply:
I agree, the static linking seems to be an issue for fairly narrow case:
   - the library must be using traditional `_fini()` entry point for its destructor
   - the library must be statically linked
if both conditions are met then destructor `_fini()` is not called at least on some OSes..

Reply:
Please can you check Windows too?